### PR TITLE
Add new term paroxysmal atrial fibrillation

### PR DESCRIPTION
**Pull request by twhetzel**
closes https://github.com/monarch-initiative/mondo/issues/7886

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -584963,6 +584963,21 @@ is_a: MONDO:0003767 {source="NCIT:C50888", source="https://orcid.org/0009-0009-6
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8003" xsd:anyURI
 
+[Term]
+id: MONDO:1030011
+name: paroxysmal atrial fibrillation
+def: "A disorder characterized by intermittent atrial fibrillation that lasts less than seven days from onset." [https://orcid.org/0009-0009-6791-365X]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:584969]: Definition:
This definition implies that "any disorder" that has continuous atrial fibrillation that lasts for more than seven days is a "persistent atrial fibrillation". This is not correct.
The definition (genus-differentia) should use the most specific genus, which here should be "atrial fibrillation" (ie the parent term).

dbxref:

Please add the PMID of the paper shared in the issue as a dbxref for this term.
this ORCID refers to a private record. Is this the correct orcid? If so, who's orcid is it?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -584963,6 +584963,21 @@ is_a: MONDO:0003767 {source="NCIT:C50888", source="https://orcid.org/0009-0009-6
...
+synonym: "paroxymsal AFib" EXACT [NCIT:C80391]
+synonym: "paroxysmal AF" EXACT [NCIT:C80391]
+xref: ICD10CM:I48.0 {source="MONDO:equivalentTo"}
+xref: icd11.foundation:542703670 {source="MONDO:equivalentTo"}
+xref: MESH:D001281 {source="MONDO:relatedTo"}
+xref: NCIT:C80391 {source="MONDO:equivalentTo"}
+xref: SCTID:282825002 {source="MONDO:equivalentTo"}
+is_a: MONDO:0004981 {source="https://orcid.org/0009-0009-6791-365X"} ! atrial fibrillation
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:584977]: Please use the PMID as a source, and/or some of the x-ref (as appropriate)
Also same question as above about the orcid

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
Please see comments

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:584969]: updated

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:584977]: updated

**Review by sabrinatoro — APPROVED** [review]

**Pull request merged**